### PR TITLE
Lab_Work_2_Dame_Stoilovski

### PR DESCRIPTION
**Pull request by DameStoilovski99**

### CODE REVIEW

File: LabWork2_Dame_Stoilovski/Lab_Work_2_Dame_Stoilovski.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from country
+where Continent in ('Asia', 'Africa') and Population > 100000000;
+
+select *
+from country
+where Name in ('Andorra', 'Barbados', 'Yemen', 'Vietnam', 'Poland');
+
+select *
+from country
+where GovernmentForm is not null and Continent not in ('South America', 'North America');
```
Comment (sslavova) [inline LabWork2_Dame_Stoilovski/Lab_Work_2_Dame_Stoilovski.sql:18]: -- Update the GovernmentForm condition - you need all republic government forms

Comment (DameStoilovski99) [reply LabWork2_Dame_Stoilovski/Lab_Work_2_Dame_Stoilovski.sql]: Okay, I fixed it now.

**Review by sslavova — CHANGES_REQUESTED** [review]
Update GovernmentForm condition on line 18 - you should take only government forms that are republic. 
Everything else is good :)

**Comment by DameStoilovski99** [comment]
> 

Didn't realize that.. :D
I think everything is fine now..

**Review by sslavova — APPROVED** [review]

**Pull request merged**